test/docker-tests.robot

```diff
@@ -163,6 +163,28 @@ Two node compose example with node joining enode group
   ...  Delete compose from  ./compose-test-6.yaml
   ...  AND  Delete compose from  ./compose-test-7.yaml
   
+Two node compose example with bootstrap node joining trying to itself
+  Start compose from  ./compose-test-8.yaml
+  Verify response for unauthenticated request with  7100  *Unauthorized*
+  Verify response for unauthenticated request with  7101  *Unauthorized*
+  Verify response for unauthenticated request with  7102  *Unauthorized*
+  Verify response for unauthenticated request with  7200  *Unauthorized*
+  Verify response for unauthenticated request with  7201  *Unauthorized*
+  Verify response for unauthenticated request with  7202  *Unauthorized*
+  Verify response for authenticated request with  7100  *Query Console*
+  Verify response for authenticated request with  7101  *No license key has been entered*
+  Verify response for authenticated request with  7102  *Monitoring Dashboard*
+  Compose logs should contain  ./compose-test-8.yaml  *bootstrap*TZ is defined, setting timezone to America/Los_Angeles.*
+  Compose logs should contain  ./compose-test-8.yaml  *bootstrap*MARKLOGIC_ADMIN_PASSWORD is set, using ENV for admin password.*
+  Compose logs should contain  ./compose-test-8.yaml  *bootstrap*MARKLOGIC_INIT is true, initializing the MarkLogic server.*
+  Compose logs should contain  ./compose-test-8.yaml  *bootstrap*MARKLOGIC_JOIN_CLUSTER is false or not defined, not joining cluster.*
+  Compose logs should contain  ./compose-test-8.yaml  *node2*TZ is defined, setting timezone to America/Los_Angeles.*
+  Compose logs should contain  ./compose-test-8.yaml  *node2*MARKLOGIC_ADMIN_PASSWORD is set, using ENV for admin password.*
+  Compose logs should contain  ./compose-test-8.yaml  *node2*MARKLOGIC_INIT is true, initializing the MarkLogic server.*
+  Compose logs should contain  ./compose-test-8.yaml  *node2*HOST cannot join itself, please check the value for MARKLOGIC_BOOTSTRAP_HOST.*
```

Comment:
to update this test when the error message is changed. thanks

Reply:
updated tests with correct error message.